docs/reference/commands.md

```diff
@@ -1147,7 +1147,14 @@ Create delete and list branches within a lakeFS repository
 Create a new branch in a repository
```

Comment:
This file is auto generated by this: https://github.com/treeverse/lakeFS/blob/master/cmd/lakectl/cmd/docs.go, please move the examples there :)

Reply:
Forget about it, you added things at the right place I got confused.